MapleServer2/Enums/Job.cs

```diff
@@ -17,22 +17,38 @@ public enum Job
         GameMaster = 999
     }
 
-    public enum JobCode
+    public enum JobCode : int
     {
         None = 0,
         Knight = 101,
-        Berserker = 201,
-        Wizard = 301,
-        Priest = 401,
-        Archer = 501,
-        HeavyGunner = 601,
-        Thief = 701,
-        Assassin = 801,
-        Runeblade = 901,
-        Striker = 1001,
-        SoulBinder = 1101,
+        Berserker = 102,
```

Comment:
this way of coding it and where you add 400 to the job id in SkillMetaData seems very confusing even though the ids seem to work. If anything this should be labelled only for skills

Reply:
True, it was for testing, ill change it on the next commit